javascript/sdk/package.json

```diff
@@ -2,22 +2,17 @@
   "name": "@theprelude/sdk",
   "version": "1.5.8",
   "type": "module",
-  "files": [
-    "dist"
-  ],
-  "main": "./dist/sdk.umd.cjs",
-  "module": "./dist/sdk.js",
-  "types": "./dist/main.d.ts",
+  "main": "lib/main.ts",
+  "types": "lib/main.ts",
   "exports": {
     ".": {
-      "import": "./dist/sdk.js",
-      "require": "./dist/sdk.umd.cjs"
+      "import": "./lib/main.ts",
```

Comment:
this file is the only concern of mine with the use of the bot @kenrick . We don't want these changed, I wonder if we can set a rule

Reply:
I think its ok until the next publishing